funnel/templates/space.html

```diff
@@ -1,6 +1,7 @@
 {% extends "layout.html" %}
 {% from "macros.html" import rsvpform, rsvpscript %}
 {% block title %}{{ space.title }}{% endblock %}
+{% block description %}{{ space.title }} &ndash; {% if g.profile %}{{ g.profile.description|striptags }}{% endif %}{% endblock %}
```

Comment:
You shouldn't have a trailing dash (in case `g.profile` is false). The dash should be inside the if block.